charts/pega/values.yaml

```diff
@@ -68,6 +69,10 @@ global:
       apiKey:
         headerName: "YOUR_ARTIFACTORY_DEDICATED_APIKEY_HEADER"
         value: "YOUR_ARTIFACTORY_API_KEY"
+      # To avoid exposing basic.username,basic.password,apiKey.headerName,apiKey.value parameters, configure the
+      # basic.username,basic.password,apiKey.headerName,apiKey.value parameters in External Secrets Manager, and enter the external secret for the credentials
+      # make sure the keys in the secret should be CUSTOM_ARTIFACTORY_USERNAME , CUSTOM_ARTIFACTORY_PASSWORD , CUSTOM_ARTIFACTORY_APIKEY_HEADER , CUSTOM_ARTIFACTORY_APIKEY
+      external_secret_name: ""
```

Comment:
Can this be changed to just `secretName`? Use came case for the values.yaml entries. Applies to other files as well.

Reply:
Discussed with Team on this, Hence resolving

to maintain backward compatibility for existing usages of external_secret_name & consistency of usage , for now we will be proceeding with  external_secret_name.

This will be handled as part of next major release